src/main/resources/liquibase/crop_changelog/20_3_0.xml

```diff
@@ -30,14 +30,46 @@
 				<constraints nullable="false"/>
 			</column>
 			<column name="created_by" type="int" defaultValue="0">
-				<constraints nullable="false" />
+				<constraints nullable="false"/>
 			</column>
 			<column name="modified_date" type="timestamp">
 				<constraints nullable="true"/>
 			</column>
 			<column name="modified_by" type="int">
-				<constraints nullable="true" />
+				<constraints nullable="true"/>
 			</column>
 		</createTable>
 	</changeSet>
+	<changeSet author="aldrin" id="v20.3.0-2">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">select count(*) from cvterm where cvterm_id = 1181;</sqlCheck>
+		</preConditions>
+		<comment>Add cvterm record for Summary Statistics Dataset</comment>
```

Comment:
As discussed, we can use cvterm_id = 1180 instead with definition "Summary type of experiment"